lnrpc/rpc.proto

```diff
@@ -1900,9 +1937,41 @@ message ChanPointShim {
     uint32 thaw_height = 6;
 }
 
+message PsbtShim {
+    /**
+    The size of the output to be created as the channel point for this channel
+    funding.
+    */
+    int64 amt = 1;
```

Comment:
Is this needed given that the greater request already has the amount specified?

Reply:
No, removed.